### PR TITLE
Add configuration options for parsing alternative input tokens and full-width unicode character support.

### PR DESCRIPTION
**Pull request by dpvc**
This is the companion PR to openwebwork/pg#518 that addresses the issues in openwebwork/pg#517.  This PR adds the needed course configuration parameters to allow course-wide control of the new features.

### CODE REVIEW

**Review by drdrew42 — CHANGES_REQUESTED** [review]
Trivial changes needed. Typos 'copyihng' and 'Fullk' in both changed files.

**Comment by dpvc** [comment]
@drdrew42, thanks!  I have fixed the typos.

**Review by drdrew42 — APPROVED** [review]

**Review by taniwallach — APPROVED** [review]
I tested this with https://github.com/openwebwork/pg/pull/518 and these settings work as intended, as far as I tested in terms of enabling/disabling support for:
  - Unicode minus and Unicode infinity
  - Full width open/close parenthesis (https://github.com/openwebwork/pg/issues/517#issuecomment-757404101)

Thanks @dpvc !!!

**Comment by dpvc** [comment]
@taniwallach, thanks for the review.

**Review by drgrice1 — APPROVED** [review]
This looks good.

**Review by Alex-Jordan — APPROVED** [review]

File: conf/defaults.config [hunk 1]
```diff
@@ -1199,6 +1199,15 @@ $pg{specialPGEnvironmentVars}{onTheFlyImageSize}      = 400,
...
 $pg{specialPGEnvironmentVars}{useOldAnswerMacros} = 0;
 
+# Determines whether or not MathObjects contexts will parse the alternative tokens
+# listed in the "alternatives" property (mostly for unicode alternatives for parse tokens).
+$pg{specialPGEnvironmentVars}{parseAlternatives} = 0;
+
+# Determines whether or not the MathObjects parser will convert the Full Width Unicode block
+# (U+FF01 to U+FF5E) to their corresponding ASCII characters (U+0021 to U+007E) automatically.
+$pg{specialPGEnvironmentVars}{convertFullWidthCharacters} = 0;
+
```
[the remark this answers is not in the document]
Comment (Alex-Jordan) [reply conf/defaults.config:1209]: Good idea. I added a commit with that.

**Comment by Alex-Jordan** [comment]
Everyone approved this, and nearly everyone approved the corresponding https://github.com/openwebwork/pg/pull/518. So I'm going to merge. I'm trying to make a dent in the PR backlog this week.

**Pull request merged**